packages/@sanity/core/src/commands/graphql/deployGraphQLAPICommand.js

```diff
@@ -3,13 +3,19 @@ import lazyRequire from '@sanity/util/lib/lazyRequire'
 const helpText = `
 Options
   --dataset <dataset> Deploy API for the given dataset
+  --tag <tag> Deploy API to given tag (defaults to 'default')
+  --generation <generation> API generation to deploy
```

Comment:
Since we now default to the latest generation, it should also be noted in the parameter here that is the default.

Reply:
Fixed!